src/System.Data.SqlClient/tests/ManualTests/SQL/SqlNamedPipesTest/SqlNamedPipesTest.cs

```diff
@@ -15,51 +15,43 @@ public static void ValidConnStringTest()
             SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(DataTestUtility.NpConnStr);
             builder.ConnectTimeout = 5;
 
-            string plainConnString = builder.ConnectionString;
-            builder.DataSource = "np:" + GetHostFromDataSource(builder.DataSource);
-            string serverNameOnlyConnString = builder.ConnectionString;
-
             if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                OpenBadConnection<PlatformNotSupportedException>(plainConnString);
-                OpenBadConnection<PlatformNotSupportedException>(serverNameOnlyConnString);
+                OpenBadConnection<PlatformNotSupportedException>(builder.ConnectionString);
             }
             else
             {
-                OpenGoodConnection(plainConnString);
-                OpenGoodConnection(serverNameOnlyConnString);
+                OpenGoodConnection(builder.ConnectionString);
             }
         }
 
 #if MANAGED_SNI
         [CheckConnStrSetupFact]
```

Comment:
Do you want to change this conditional Fact attribute to [Fact] ? I think this test can be executed without any server.